### ISSUE TITLE
Charge problems with 12000maH battery

### ISSUE THREAD

**Issue by serge16** [PiSupply/PiJuice#315]
I saw on this forum several issues concerning the battery charge. I also have some problems.
For almost 2 months I have been testing a Pijuice connected to a Raspberry 3B + and an official 7 "screen and a 12000 mAh battery. With the original firmware the battery gauge was giving anything, I installed the beta version 28- 02 and the gauge worked better.

The first problem is that there is no predefined profile for the 12000mah battery, even with this beta version.
Creating a custom profile with pijuice_cli crashes when APPLY is done. With pijuice_gui when changing values ​​they return to their initial values ​​when doing apply. For example if we change "Termination current" by 600 after APPLY, 400 come back.

I had to use a Python script found on this forum (https://github.com/PiSupply/PiJuice/issues/305) to successfully set the profile.

The second problem is more important. At the beginning of my tests when the battery was fully charged, when I unplugged the power supply, the voltage was falling to something slightly below 4V (3.949 for example). And the battery was discharging for + 10H (screen on half time, 30% brightness and no activity). Then, I reloaded the battery and so on.
For some time (I do not know when or why) the battery is not charging. After long hours and even days of charging the displayed voltage (unplugged power supply) does not exceed about 3,750.
Sometimes I find the behavior of the beginning and the battery is fully charged. It seems to me that this is after disconnecting the power supply and the battery for a few minutes, but I'm not sure.

In short, I spent many hours to understand but it seems obvious that there is really a problem of battery charge.

**Comment by ChristopherRush** [comment]
I believe these issues have been resolved with updates.
